setup.sh

```diff
@@ -43,9 +43,18 @@ done
 url=${url:-http://localhost}
 printf "URL: ${url}\n\n"
 
+jwt_secret=$(tr -cd "[:alnum:]" < /dev/urandom | head -c 42 | xargs -0)
```

Comment:
when I ran script I got the following output
<img width="257" alt="image" src="https://user-images.githubusercontent.com/6690616/178707070-d44a9ee7-d647-41f4-af77-070b413c82c7.png">
and my variables had empty values
<img width="367" alt="image" src="https://user-images.githubusercontent.com/6690616/178707242-9aea359c-c604-4033-82e9-36746e472b57.png">

Reply:
Fixed